### DISCUSSION TITLE
对类型“Cache”的引用声称该类型是在“System.Web”中定义的

### DISCUSSION THREAD

**Discussion by wh411810263** [ldqk/Masuit.Tools#111] [Q&A]
我用nuget包导入后使用CacheHelper类显示这个错误，CS7069对类型“Cache”的引用声称该类型是在“System.Web”中定义的,webapi，使用的版本是2023.3.4的版本

**Comment by ldqk** [comment]
请前往https://github.com/ldqk/Masuit.Tools/issues/new/choose 提交issue
